dts/arm/Makefile

```diff
@@ -1,4 +1,9 @@
 ifeq ($(CONFIG_HAS_DTS),y)
+
+ifdef CONFIG_SOC_FAMILY_STM32
+ZEPHYRINCLUDE += -I$(srctree)/dts/arm/st
```

Comment:
I had a try but didn't work. Maybe I was not lucid anymore but is there any subtlety to achieve this?
I guess #include "dts/arm/st/dt_stm32_clocks.h" would be more likely to work